csxtools/utils.py

```diff
@@ -224,10 +224,10 @@ def get_fastccd_timestamps(header):
         list of arrays of the timestamps
 
     """
-    hover = {'fccd_image_lightfield': AreaDetectorHDF5TimestampHandler}
-    img = [i for i in get_events(header, ['fccd_image_lightfield'],
+    hover = {tag: AreaDetectorHDF5TimestampHandler}
+    img = [i for i in get_events(header, [tag],
                                  handler_overrides=hover)]
-    timestamps = img[0]['data']['fccd_image_lightfield']
+    timestamps = [i['data'][tag] for i in img if tag in i['data'].keys()]
```

Comment:
This will silently drop images that do not have a time stamp which may make it difficult to realign the timestamps with the images later.  Maybe something like

```python
timestamps = [i['data'][tag] if tag in i['data'].keys() else np.nan for i in img]
```

which will fill in `np.nan` for the missing values.  Or, as these are dictionaries coming back

```python
timestamps = [d['data'].get(tag, np.nan) for d in img]
```